Assets/Scripts/InputModule/Processor/PCInputProcessor.cs

```diff
@@ -18,6 +18,9 @@ public class PCInputProcessor: InputProcessor {
 
 			binding.InfographicToggle.OnPress += () => Controller.GraphController.Infographic.SetActive(!Controller.GraphController.Infographic.activeSelf);
 			binding.ConnectionModeToggle.OnPress += () => Controller.GraphController.SwitchConnectionMode();
+
+            binding.Undo.OnPress += Undo;
```

Comment:
- Add Cave buttons with the same bindings
- Assign them according to our specification